Automattic/wp-calypso
# Add stats to quantity dropdown

## Proposed Changes

* Add stats alias constants
* Add stats quantity dropdown to product lightbox

## Testing Instructions

1. Using the Calypso live link, go  to the `/pricing/` page
2. On the stats card, select the "Get" CTA
3. Make sure the lightbox opens with a dropdown instead of going to checkout
![image](https://github.com/Automattic/wp-calypso/assets/65001528/4ee0a31b-7bb6-4bb5-9c68-f0f5207f97a2)
4. Switch between the tiers and make sure the request limit in the includes list and the price is updated
![Screenshot 2024-05-09 at 11 12 46 AM](https://github.com/Automattic/wp-calypso/assets/65001528/79c3dd54-6a45-4bf3-8483-0999e12b9e2e)
5. Click `Proceed to Checkout` and make sure the correct quantity gets put into the cart
![image](https://github.com/Automattic/wp-calypso/assets/65001528/790eff25-dbc6-43bd-85ec-63f1a9b0e2d4)

## Pre-merge Checklist
